updatecli-manifest.yaml

```
@@ -0,0 +1,44 @@
+title: Update versions.yaml file
+
+scms:
+  pluto-git:
+    kind: git
+    spec:
+      url: https://github.com/FairwindsOps/pluto.git
+      branch: master
+
+  policy-git:
+    kind: github
+    spec:
+      owner: "kubewarden"
+      repository: "deprecated-api-versions-policy"
+      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "GITHUB_AUTHOR" }}'
```

```suggestion
      username: '{{ requiredEnv "GITHUB_ACTOR" }}'
```
If you want to reuse the environment set by the Github Action, I think you need to use GITHUB_ACTOR
https://docs.github.com/en/actions/learn-github-actions/environment-variables